salt/cli/batch.py

```diff
@@ -103,7 +103,7 @@ def __update_wait(self, wait):
         if i:
             del wait[:i]
 
-    def run(self):
+    def run(self, safe_batch = False):
```

Comment:
I couldn't find a way to let run() know it's being forced into a batch mode without actually passing the value as a parameter because the options are re-read shortly before this line. I'd be happy to implement a better option, but I'm not currently aware of what that might be.